tools/slang-internals-test/slang-internals-test-main.cpp

```
@@ -0,0 +1,138 @@
+
+    const char* m_currentTestName = "<unknown>";
+    bool m_currentTestFailed = false;
+};
+
+} // namespace
+
+extern "C" IUnitTestModule* slangUnitTestGetModule();
+
+int main(int argc, char** argv)
+{
+    // Optional substring filter, so a developer can run a single test while
```

Cppcheck (2.21.0)

[style] The function 'FastFloor' is never used.

(unusedFunction)